docs/plugins/analytics.md

```diff
@@ -55,7 +57,7 @@ learn how to contribute the integration yourself!
 [int-howto]: #writing-integrations
 [analytics-api-type]: https://backstage.io/docs/reference/core-plugin-api.analyticsapi
```

Comment:
Could you please add the link to the module readme file?
Here is and example: https://github.com/backstage/backstage/pull/20918/files